Corrections/triggerlookup.cc

```diff
@@ -1,36 +1,39 @@
+//   MUST MAKE TRIGGER WEIGHTS BEFORE USE!!!
+
 #include <cmath>
 using namespace ROOT::VecOps;
-using rvec_f = const RVec<float> &;
-using rvec_i = const RVec<int> &;
 
 namespace analyzer {
-    std::vector<float> Trigger_Lookup(rvec_f var, TH1F* TRP ){
-        float Weight = 1.0;
-        float Weightup = 1.0;
-        float Weightdown = 1.0;
+    std::vector<float> TriggerLookup(float var, TH1D* TRP ){
+        float Weight = 1;
+        float Weightup = 1;
+        float Weightdown = 1;
 
         std::vector<float> out;
 
         if (var < 2000.0){
```

Comment:
This 2000 cut is a left-over from Kevin's code and sort-of arbitrary (especially since someone could make a trigger efficiency on any variable and 2000 may be too high or too low). Fine for now but we should add an argument that gives the user the ability to change the threshold and it defaults to no threshold